src/contributors/02-local-development.md

```diff
@@ -90,6 +90,16 @@ Use `cargo check` to find compilation errors. To start the Lemmy backend, run `c
 
 After making changes, you need to format the code with `cargo +nightly fmt --all` and run the linter with `./scripts/lint.sh`.
 
+If you'd like to speed up compilation times, you can add the following to `~/.cargo/config.toml` (requires nightly rust).
+
+```toml
+[unstable]
+codegen-backend = true
```

Comment:
From the docs it sounds like this has no effect at all on its own.

https://doc.rust-lang.org/cargo/reference/unstable.html#codegen-backend